database/postGres/galleryPosts_datagenerator.js

```diff
@@ -4,65 +4,54 @@ const argv = require('yargs').argv
 const faker = require('faker');
 const { image } = require('faker');
 
-const lines = argv.lines || 10;
+// const lines = argv.lines || 10;
 const filename = argv.output || 'galleryPosts.csv'
 const writeStream = fs.createWriteStream(filename);
-let listingID = 10000000;
+let listingID = 10000001;
+let galleryNum = 0;
 
 let createPost = () => {
     //need to fix this function and whole createpost to render randomlisting_id and gallery url 
+    if (galleryNum === 0) {
+        listingID--;
+        galleryNum = Math.floor(Math.random() * (15 - 5 + 1)) + 5; 
+      }; 
 
     let imageNum = (Math.floor(Math.random() * (1000 - 1 + 1)) + 1).toString().padStart(4, 0);
 
     //find way to pull s3 url into gallery that is randomized
-    const listing_id = listingID;
-    const gallery = `https://tripadvisor-carousel-dump.s3-us-west-2.amazonaws.com/galleryImages/${imageNum}.jpg;`
-    // galleryNum--;
+    // const listing_id = listingID;
+    const gallery = `https://tripadvisor-carousel-dump.s3-us-west-2.amazonaws.com/galleryImages/${imageNum}.jpg`;
+    galleryNum--;
 
-    return `${listing_id},${gallery}\n`
+    return `${listingID},${gallery}\n`
 
 };
 
 const startWriting = (writeStream, encoding, done) => {
-    // let i = lines;
-    // let listingID = 11;
-    let galleryNum = 0;
+
 
     function writing () {
       let canWrite = true;
       
       do {
-        // i--;
-        if (galleryNum === 0) {
-            listingID--;
-            galleryNum = Math.floor(Math.random() * (15 - 5 + 1)) + 5; 
-          }; 
-
+        if (listingID % (Math.floor(10000000 / 10)) === 1000) {
```

Comment:
may want to make more generalizable for different amounts of listings you're creating, but not necessary